### PR TITLE
#14 add typings on sjs module

### PR DESCRIPTION
**Pull request by lucagez**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * @default regex - \\n|\\r|\\t|\\"|\\\\
+ */
+export function escape(regex?: RegExp): (str: string) => string;
+/**
+ * sjs
+ * 
+ * compile the provided schema and exports a function
+ * that serialize objects enforcing the schema.
+ */
+export function sjs(schema: SjsSchema): SjsEscaper;
```
Comment (markmcdowell) [inline index.d.ts:77]: Typo line 77 it should be: 
`export function sjs(schema: SjsSchema): SjsSerializer;`

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * Usage: https://github.com/lucagez/slow-json-stringify#usage
+ */
+export type SjsSchema = { [prop: string]: AttrExecutable | SjsSchema };
+
+/**
+ * SjsSerializer
+ * 
+ * Serialize any object enforcing the provided schema.
+ */
+export type SjsSerializer = (obj: object) => string;
```
Comment (markmcdowell) [inline index.d.ts:30]: It would be better to use unknown here instead of object.
`export type SjsSerializer = (obj: unknown) => string;`

**Review by markmcdowell — COMMENTED** [review]
The schema works with the 2 changes mentioned. Would be nice to have it merged! 👍

**Comment by lucagez** [comment]
@markmcdowell Thanks for your review! Change has been merged and published as `2.0.1`

**Pull request merged**